### PR TITLE
Add RPC to retrieve single quote

### LINKED ISSUE #898: RPC to get the OE report of the node the user is connected to
**Is your feature request related to a problem? Please describe.**
Currently there is an RPC for the user get a list of OE reports from all the CCF nodes.  It would be nice to have an RPC to get the OE report for the current node the user is connected to.

**Describe the solution you'd like**
The user would connect to a CCF node and request, via this new RPC, the OE report of the connected node.

**Describe alternatives you've considered**
It is possible to use the current set of RPCs, to get all the reports and the current node id, but that would be a little bit more work.

### PR DESCRIPTION
**Pull request by eddyashton**
Resolves #898.

### CODE REVIEW

**Comment by cimetrics** [comment]
node_quote_rpc2@[5475 aka 20200302.12](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=5475&view=results) vs master ewma over 15 builds from [5233](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=5233&view=results) to [5456](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=5456&view=results)
images

File: sphinx/source/schemas/getQuote_result.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
```
Comment (jumaffre) [inline sphinx/source/schemas/getQuote_result.json]: Should this and `getQuotes` be added to `users/rpc_api.rst` in the `Common Methods` section?

Comment (eddyashton) [reply sphinx/source/schemas/getQuote_result.json]: I don't think so, as they're currently only available on the nodes frontend?

**Comment by jazzybluesea** [comment]
Thank you!

**Pull request merged**